packages/uicore/src/components/Accordion/Accordion.tsx

```diff
@@ -46,9 +47,14 @@ function AccordionPanel(
     summaryClassName,
     detailsClassName,
     chevronClassName,
-    className
+    className,
+    shouldRender
   } = props
 
+  if (shouldRender && !shouldRender?.()) {
+    return <></>
```

Comment:
we can return `null` instead to be more clear

Reply:
Done